src/main/java/com/echobox/api/linkedin/connection/versioned/VersionedOrganizationConnection.java

```diff
@@ -178,6 +184,28 @@ public List<OrganizationBrand> retrieveOrganizationBrandByParentOrganization(
         parameters.toArray(new Parameter[0]));
   }
   
+  /**
+   * Find a Member's Organization Access Control Information
+   * E.g. https://api.linkedin.com/rest/organizationAcls?q=roleAssignee
+   * @see <a href="https://learn.microsoft.com/en-us/linkedin/marketing/integrations/community-management/organizations/organization-access-control-by-role?view=li-lms-2022-11&tabs=http">
+   * Access Control</a>
+   * @param role Limit results to specific roles, such as ADMINISTRATOR.
+   * @param state Limit results to specific role states, such as APPROVED.
+   * @param projection Field projection
+   * @param count the number of entries to be returned per paged request
+   * @return List of access controls for a given role and state for the member
+   */
+  public List<AccessControl> fetchMemberOrganizationAccessControl(String role, String state,
```

Comment:
The method names above seem to start with `retrieve...`. If they are doing equivalent things then please can the name be updated.